Adyen/UI/Form/Items/Value/FormValueItemView.swift

```diff
@@ -91,41 +90,58 @@ open class FormValueItemView<ValueType, Style, ItemType: FormValueItem<ValueType
     }()
     
     internal var defaultSeparatorColor: UIColor {
-        if isFirstResponder {
+        if isEditing {
             return tintColor
         } else {
             return item.style.separatorColor ?? UIColor.Adyen.componentSeparator
         }
     }
     
+    internal var defaultTitleColor: UIColor {
+        if isEditing {
+            return tintColor
+        } else {
+            return item.style.title.color
+        }
+    }
+    
     internal func highlightSeparatorView(color: UIColor) {
         let transitionView = UIView()
         transitionView.backgroundColor = color
         transitionView.frame = separatorView.frame
         transitionView.frame.size.width = 0.0
         addSubview(transitionView)
         
-        UIView.animate(withDuration: 0.25, delay: 0.0, options: [.curveEaseInOut], animations: {
-            transitionView.frame = self.separatorView.frame
-        }, completion: { _ in
-            self.separatorView.backgroundColor = color
-            transitionView.removeFromSuperview()
-        })
+        let context = AnimationContext(animationKey: separatorHighlightingAnimationKey,
+                                       duration: 0.25,
+                                       delay: 0.0,
+                                       options: [.curveEaseInOut],
+                                       animations: {
+                                           transitionView.frame = self.separatorView.frame
+                                       },
+                                       completion: { _ in
+                                           self.separatorView.backgroundColor = color
+                                           transitionView.removeFromSuperview()
+                                       })
+        
+        animate(context: context)
     }
     
+    private let separatorHighlightingAnimationKey = "separator_highlighting"
+    
     internal func unhighlightSeparatorView() {
-        let transitionView = UIView()
-        transitionView.backgroundColor = tintColor
-        transitionView.frame = separatorView.frame
-        addSubview(transitionView)
+        let context = AnimationContext(animationKey: separatorHighlightingAnimationKey,
+                                       duration: 0.0,
```

Comment:
animation with duration 0 is no animation.. do we still need it as `animate(context: context)` ?

Reply:
yeah, that's by intention, I wanted it to not be animated, but also wanted the synchronization with the highlighting :-) .